dockers-apt-repo.sh

```diff
@@ -1,17 +1,40 @@
-# this is just notes going to automate!
-# Add Docker's official GPG key:
-sudo apt-get update
-sudo apt-get install ca-certificates curl
-sudo install -m 0755 -d /etc/apt/keyrings
-sudo curl -fsSL https://download.docker.com/linux/debian/gpg -o /etc/apt/keyrings/docker.asc
-sudo chmod a+r /etc/apt/keyrings/docker.asc
 
-# Add the repository to Apt sources:
+# Creating encryption certificates
+if (  which ca-certificates curl )
```

Comment:
This is 2 different programs, why not check for each individually?

Reply:
I thought so too, but from the docs It looks like an argument.
https://docs.docker.com/engine/install/debian/

Reply:
when in doubt test it in the terminal... which whatever